music_assistant/providers/spotify_connect/__init__.py

```diff
@@ -213,7 +215,7 @@ async def _librespot_runner(self) -> None:
                 "--volume-ctrl",
                 "fixed",
                 "--initial-volume",
-                "100",
+                f"{self.player.volume_level if self.player else 100}",
```

Comment:
Note that volume_level can be None for players that do not support volume control so this should be a little more gracefully handled.

Reply:
Thanks for pointing this out. All my media players have volume, so I wasn't sure how that would behave. Updated👍